src/Document/Single.php

```diff
@@ -54,7 +55,7 @@ class Document extends Element
      *
      * @param mixed $id
      *
-     * @return Document
+     * @return $this
```

Comment:
I'm sure the doc block should state the class name, and not ```$this```. 

https://github.com/phpDocumentor/phpDocumentor2/issues/800